viya_ark_library/k8s/test_impl/response_data/resources_pgclusters.webinfdsvr.sas.com.json

```diff
@@ -0,0 +1,202 @@
+[
+    {
+        "apiVersion": "webinfdsvr.sas.com/v1",
+        "kind": "Pgcluster",
+        "metadata": {
+            "annotations": {
+                "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"webinfdsvr.sas.com/v1\",\"kind\":\"Pgcluster\",\"metadata\":{\"annotations\":{\"sas.com/default-database\":\"true\"},\"labels\":{\"sas.com/admin\":\"cluster-local\",\"sas.com/deployment\":\"sas-viya\"},\"name\":\"postgres\",\"namespace\":\"d36363\",\"ownerReferences\":[{\"apiVersion\":\"orchestration.sas.com/v1alpha1\",\"blockOwnerDeletion\":true,\"controller\":true,\"kind\":\"SASDeployment\",\"name\":\"sas-viya\",\"uid\":\"74904415-de32-497f-9d5e-1b06c7058305\"}]},\"spec\":{\"certframe\":{\"containers\":[{\"envFrom\":[{\"configMapRef\":{\"name\":\"sas-tls-config-f8ccd48c6m\"}}],\"name\":\"ignoreMe\",\"volumeMounts\":[{\"mountPath\":\"/security\",\"name\":\"security\"}]}],\"initContainers\":[{\"env\":[{\"name\":\"KUBE_POD_NAME\",\"valueFrom\":{\"fieldRef\":{\"fieldPath\":\"metadata.name\"}}},{\"name\":\"SAS_CERTFRAME_TOKEN_DIR\",\"value\":\"/certframe-token\"},{\"name\":\"SAS_ADDITIONAL_CA_CERTIFICATES_DIR\",\"value\":\"/customer-provided-ca-certificates\"}],\"envFrom\":[{\"configMapRef\":{\"name\":\"sas-certframe-ingress-certificate-config-cmm2t44t88\"}},{\"configMapRef\":{\"name\":\"sas-certframe-user-config-c4ch2c59m7\"}},{\"configMapRef\":{\"name\":\"sas-certframe-config-2ch97fd95b\"}}],\"image\":\"test.sas.com/test/sas-certframe:3.17.5-20210729.1627580385314\",\"name\":\"sas-certframe\",\"resources\":{\"limits\":{\"cpu\":\"500m\",\"memory\":\"500Mi\"},\"requests\":{\"cpu\":\"50m\",\"memory\":\"50Mi\"}},\"securityContext\":{\"allowPrivilegeEscalation\":false,\"capabilities\":{\"drop\":[\"ALL\"]},\"privileged\":false,\"readOnlyRootFilesystem\":true},\"volumeMounts\":[{\"mountPath\":\"/certframe-token\",\"name\":\"certframe-token\"},{\"mountPath\":\"/security\",\"name\":\"security\"},{\"mountPath\":\"/customer-provided-ca-certificates\",\"name\":\"customer-provided-ca-certificates\"}]}],\"volumes\":[{\"name\":\"certframe-token\",\"secret\":{\"secretName\":\"sas-certframe-token\"}},{\"emptyDir\":{},\"name\":\"security\"},{\"configMap\":{\"name\":\"sas-customer-provided-ca-certificates-km44fc867t\"},\"name\":\"customer-provided-ca-certificates\"}]},\"connection\":{\"host\":\"testm1.openstack.sas.com\",\"port\":5432,\"ssl\":true},\"database\":\"SharedServices\",\"internal\":false,\"rolesecret\":\"postgres-user\"}}\n",
```

Comment:
This annotation can be deleted to reduce file size.

Reply:
Removed.